content/posts/2023/reana-0.9.0.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "REANA 0.9.0 is released"
+date: 2023-01-24T09:00:00+01:00
+---
+
+We are happy to announce REANA 0.9.0, a major update that allows
+users to set custom workflow retention rules to automatically delete workspace
+files, adds a new way of launching workflows from external sources, and more!
+
+<!--more-->
+
+## What's new for the users?
+
+### Automatically delete workspace files using custom retention rules
+
+It's helpful to have a way to automatically delete files in your workflow's
+workspace that you don't need anymore. For example, if your workflow creates
+large temporary files that take up a lot of space that consume your
+[disk quota](https://docs.reana.io/advanced-usage/user-quotas), you may want to
+get rid of them after a certain period of time. Therefore, we're introducing the
+ability to set up custom rules for keeping or deleting files in your workspace.
+This can save you from having to manually delete them, which can be a
```

Comment:
I would remove the last phrase "This can save you..." because it ends with "time-consuming process" evoking negative feelings. Staying positive what we have introduced should be enough here. And, for more details, there is the dedicated blog post.